tests/python/PyImportSortBearTest.py

```diff
@@ -20,4 +20,5 @@
     ('import xyz\nimport abc\n',
      'import abc\nimport xyz\n'),
     settings={'known_standard_library_imports': 'xyz',
-              'known_first_party_imports': 'abc'})
+              'known_first_party_imports': 'abc',
+              'known_third_party_imports': 'datetime'})
```

Comment:
This is not enough. You need to include `datetime` in the good and bad file as well, to see if the setting has any effect.